bokehjs/src/lib/models/plots/plot.ts

```diff
@@ -135,7 +135,10 @@ export class Plot extends LayoutDOM {
       frame_width:       [ Nullable(Number), null ],
       frame_height:      [ Nullable(Number), null ],
 
-      title:             [ Or(Ref(Title), String, Null), () => new Title({text: ""}) ],
+      // revise this when https://github.com/microsoft/TypeScript/pull/42425 is merged
+      title:             [ Or(Ref(Title), String, Null), "", {
+        convert: (title) => isString(title) ? new Title({text: title}) : title,
```

Comment:
why not `accepts` to mirror the Python side?

Reply:
I didn't think about Python here, but also it has different semantics.